packages/libhdf5/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: libhdf5
+  version: 1.12.1
+
+source:
+  sha256: e6dde173c2d243551922d23a0387a79961205b018502e6a742acb30b61bc2d5f
+  url: https://github.com/HDFGroup/hdf5/archive/refs/tags/hdf5-1_12_1.tar.gz
+
+requirements:
+  run:
+    - zlib
+
+build:
+  library: true
```

Comment:
Not sure what that means.

This is a C++ library with Python package. The primary need here is to use the Python package.